src/Bridge/Doctrine/Orm/Extension/ContextAwareQueryItemExtensionInterface.php

```diff
@@ -1,28 +0,0 @@
-<?php
```

Comment:
Deleted as it was a (wrong) duplicate of https://github.com/api-platform/core/blob/master/src/Bridge/Doctrine/Orm/Extension/QueryResultItemExtensionInterface.php, accidentally added in https://github.com/api-platform/core/pull/1619